cloud/docker/docker.py

```diff
@@ -1313,9 +1334,12 @@ def get_deployed_containers(self):
         Return any matching containers that are already present.
         """
 
+        entrypoint = self.module.params.get('entrypoint')
+        if entrypoint is not None:
+            entrypoint = shlex.split(entrypoint)
         command = self.module.params.get('command')
-        if command:
-            command = command.strip()
+        if command is not None:
```

Comment:
Is there an edge case here if `command` is an empty string? How would `shlex.split` behave with an empty string?

I guess what I'm getting at is why `if command is not None:` vs `if command:`